roles/k8s_addons/tasks/autoscaler.yml

```diff
@@ -17,14 +25,12 @@
   shell: |
     aws iam put-role-policy --role-name {{ static_nodes.stack_outputs.NodeInstanceRole.split('/')[1] }} --policy-name {{ aws_project_name }}-autoscaling --policy-document file://{{ role_path }}/files/autoscaling-policy.json
 
-- name: install autoscaler
+# Taken from https://raw.githubusercontent.com/kubernetes/autoscaler/master/cluster-autoscaler/cloudprovider/aws/examples/cluster-autoscaler-autodiscover.yaml
```

Comment:
Helm didn't add much anymore as it is also templating. This gives us more control.